### ISSUE TITLE
Feature Request:  Landscape Deformation for footprints, trails and trample

### ISSUE THREAD

**Issue by delebash** [GameDev4K/unreal-open-land#23]
Take a look at this example:
https://www.youtube.com/watch?v=HoeTIUf8Qs8&t=1s

He also has an in depth tutorial with a sample project of how he created this effect
https://www.youtube.com/playlist?list=PLNTm9yU0zou7mhiQRuEiysFqjtnizdvUM

This one has a nice feature of mud getting on your boots and then drying off
https://www.youtube.com/watch?v=uuJi5QfRiGI&t=112s

**Comment by arunoda** [comment]
Yeah. Thanks for this info. I will add this to the roadmap & some other terrain tailing effects.
As of now, you have to customize the material.  Check our [Core Customization](https://www.notion.so/OpenLand-Documentation-2268081d3b8e4a49a0d824a7ab0b7b44#06cda09bb5644a91b0bf60fce33bbd0a) guide.
